routes/donations.js

```diff
@@ -14,6 +15,19 @@ router.get('/', async (req, res) => {
   }
 });
 
+router.get('/verify', async (req, res) => {
```

Comment:
in `get` requests, the protocol doesn't allow to pass in a payload (`req.body`), so instead convert this to a `post` route (don't forget to change this in the frontend as well)